### ISSUE TITLE
Not working with Win10, Notebook in anaconda

### ISSUE THREAD

**Issue by JialongJiang** [dobestan/jupyter-notebook-themes#4]
The chrome extension is not working on my computer. Win10, 64bit, notebook is installed with anaconda environment.

**Comment by Jerzup** [comment]
Yep, same here. Works perfectly fine on my OS X Sierra but not on my Win10 Chrome laptop.

**Comment by agerolymbos** [comment]
same here :(

**Comment by forloopkilla** [comment]
not working here either on mine, someone help??

**Comment by LaureatePoet** [comment]
Win10 Chrome笔记本电脑上没用

**Comment by katiaiwasaki** [comment]
It worked for me after I reopened the notebook page and changed the theme>apply.
